debian/changelog

```diff
@@ -1,3 +1,22 @@
+ubuntu-advantage-tools (27.0.2) trusty; urgency=medium
```

Comment:
A build commit slipped in here
```suggestion
ubuntu-advantage-tools (27.0.2) impish; urgency=medium
```

Reply:
+1 dropping pending successful jenkinsrun.